.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Would weekly create an extra maintenance burden on us? Would monthly create less noise or we run into the risk of falling behind the dependencies too much?

Reply:
You are entirely right, "monthly" will do. Fixed.